www-apps/rutorrent/rutorrent-3.10.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit webapp
+
+DESCRIPTION="ruTorrent is a front-end for the popular Bittorrent client rTorrent"
+HOMEPAGE="https://github.com/Novik/ruTorrent"
+SRC_URI="https://github.com/Novik/ruTorrent/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE=""
+
+need_httpd_cgi
```

Comment:
might need to be called after DEPEND/RDEPEND?

Reply:
Correct, 
https://devmanual.gentoo.org/eclass-reference/webapp.eclass/index.html
https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/webapp.eclass#n164